### PR TITLE
feat: Events: add video enable column

### PR DESCRIPTION
**Pull request by ranjitkshah**
reqd for #https://github.com/fossasia/open-event-frontend/issues/6416

#### Short description of what this resolves:
adding video-enable column

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The unit tests pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

- [ ] All the functions created/modified in this PR contain relevant docstrings.

### CODE REVIEW

**Review by MeghalBisht — APPROVED** [review]

File: app/api/schema/events.py [hunk 1]
```diff
@@ -83,6 +83,7 @@ def validate_timezone(self, data, original_data):
     is_billing_info_mandatory = fields.Bool(default=False)
     is_donation_enabled = fields.Bool(default=False)
     is_chat_enabled = fields.Bool(default=False)
+    is_videoroom_enabled = fields.Bool(default=False)
```
Comment (MeghalBisht) [inline app/api/schema/events.py:86]:
```suggestion
    is_video_enabled = fields.Bool(default=False)
```

File: app/models/event.py [hunk 2]
```diff
@@ -62,6 +62,7 @@ class Privacy:
     is_promoted = db.Column(db.Boolean, default=False, nullable=False)
     is_demoted = db.Column(db.Boolean, default=False, nullable=False)
     is_chat_enabled = db.Column(db.Boolean, default=False, nullable=False)
+    is_videoroom_enabled = db.Column(db.Boolean, default=False, nullable=False)
```
Comment (MeghalBisht) [inline app/models/event.py:65]:
```suggestion
    is_video_enabled = db.Column(db.Boolean, default=False, nullable=False)
```

File: migrations/versions/rev-2021-06-11-00:45:55-3f0ae16c8c11_.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+
+# revision identifiers, used by Alembic.
+revision = '3f0ae16c8c11'
+down_revision = 'a739ff6a3855'
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('events', sa.Column('is_videoroom_enabled', sa.Boolean(), nullable=False, server_default='False'))
```
Comment (MeghalBisht) [inline migrations/versions/rev-2021-06-11-00:45:55-3f0ae16c8c11_.py:21]:
```suggestion
    op.add_column('events', sa.Column('is_video_enabled', sa.Boolean(), nullable=False, server_default='False'))
```

File: migrations/versions/rev-2021-06-11-00:45:55-3f0ae16c8c11_.py [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    op.add_column('events_version', sa.Column('is_videoroom_enabled', sa.Boolean(), autoincrement=False, nullable=True))
```
Comment (MeghalBisht) [inline migrations/versions/rev-2021-06-11-00:45:55-3f0ae16c8c11_.py:22]:
```suggestion
    op.add_column('events_version', sa.Column('is_video_enabled', sa.Boolean(), autoincrement=False, nullable=True))
```

File: migrations/versions/rev-2021-06-11-00:45:55-3f0ae16c8c11_.py [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+    # ### end Alembic commands ###
+
+
+def downgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_column('events_version', 'is_videoroom_enabled')
+    op.drop_column('events', 'is_videoroom_enabled')
```
Comment (MeghalBisht) [inline migrations/versions/rev-2021-06-11-00:45:55-3f0ae16c8c11_.py:29]:
```suggestion
    op.drop_column('events_version', 'is_video_enabled')
    op.drop_column('events', 'is_video_enabled')
```

**Review by MeghalBisht — DISMISSED** [review]

**Review by maze-runnar — DISMISSED** [review]

**Comment by prateekj117** [comment]
@ranjitkshah Please use proper commit messages.

**Pull request merged**